applications/launchpad/gui-react/src/containers/SettingsContainer/BaseNodeSettings/BaseNodeSettings.tsx

```diff
@@ -22,12 +25,26 @@ import MessagesConfig from '../../../config/helpMessagesConfig'
 const BaseNodeSettings = ({
   control,
   onBaseNodeConnectClick,
+  setValue,
 }: {
   control: Control<SettingsInputs>
   onBaseNodeConnectClick: () => void
+  setValue: UseFormSetValue<SettingsInputs>
 }) => {
   const theme = useTheme()
   const dispatch = useAppDispatch()
+  const selectDirectory = useCallback(async () => {
+    const selectedFolder = await open({
+      directory: true,
+      defaultPath: await appDir(),
```

Comment:
I would suggest I change here:
Now it always opens on the same directory. Even if we already used this dialog to select some directory, that new one selection is not a default path in new interaction. So we can do this:

```suggestion
      const selectDirectory = useCallback(async (lastPath?: string) => {
    const selectedFolder = await open({
      directory: true,
      defaultPath: lastPath || await appDir(),
```

... and then we need to make another change in the input...

Reply:
Done